### PR TITLE
Remaining javax-to-jakarta renames

### PR DESCRIPTION
**Pull request by dblevins**

### CODE REVIEW

File: spec/src/main/asciidoc/ServerPages.adoc [hunk 1]
```diff
@@ -11003,7 +11003,7 @@ Jakarta Server Pages specification. This appendix is non-normative.
 
 === Changes between JSP 3.0 and JSR 245
 
-* The API has moved from the `javax.servlet.jsp` package to the
+* The API has moved from the `jakarta.servlet.jsp` package to the
```
Comment (arjantijms) [inline spec/src/main/asciidoc/ServerPages.adoc:11006]: This doesn't seem entirely correct ;)

**Review by arjantijms — CHANGES_REQUESTED** [review]

File: spec/src/main/asciidoc/ServerPages.adoc (hunk 1 above)
Comment (dblevins) [reply spec/src/main/asciidoc/ServerPages.adoc:11006]: Whoops :)  Will change that back.  Tried to catch statements like those, but this one slipped through the cracks.  Thanks!

Comment (dblevins) [reply spec/src/main/asciidoc/ServerPages.adoc]: Fixed

**Review by markt-asf — APPROVED** [review]
LGTM  niw

**Review by arjantijms — APPROVED** [review]

**Pull request merged**